lib/frr_pthread.c

```diff
@@ -67,6 +68,7 @@ void frr_pthread_init()
 {
 	pthread_mutex_lock(&frr_pthread_hash_mtx);
 	{
+		stream_init_last();
```

Comment:
I think this belongs in `libfrr.c` initialization code, not here.

Reply:
moved